dev-infrastructure/modules/network/nsp-profile.bicep

```diff
@@ -0,0 +1,64 @@
+param location string
+
+@description('Access mode for this NSP')
+@allowed(['Audit', 'Enforced', 'Learning'])
```

Comment:
please remove this. EV2 precompiles the bicepparam files but leaves the placeholders in them during validation, which then fails this validation